ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml

```diff
@@ -62,6 +62,14 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: e2e-component
+  commands: |
+    if [ -d ./test/e2e ]; then
```

Comment:
I do not fancy this approach.

This is always chicken&egg but rather then faking a result for e2e which clobbers history, we have conventionally delivered first the e2e code and as it isn't used it not critical to anything yet. Then setup a CI test check the rehearsals and iterate on the osin repo e2e code if needs be. Then when the e2e rehearsal is green, merge the release PR.

Reply:
fixed